sdk/go/asymmetric_decryopt.go

```diff
@@ -0,0 +1,87 @@
+package ehsm
+
+import (
+	hmac "crypto/hmac"
+	sha256 "crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	ioutil "io/ioutil"
+	"strconv"
+	"time"
+
+	"github.com/iancoleman/orderedmap"
+)
+
+type ADecryptResponse struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+	Result  struct {
+		Plaintext string `json:"plaintext"`
+	} `json:"result"`
+}
+
+func (c *Client) AsymmetricDecrypt(keyid, data, padding_mode string) (string, error) {
```

Comment:
add comments for each interfaces, to describe more details for each params, such as type and the expected value.

Reply:
ok